### PR TITLE
rabbitmq: collect vhosts msg metrics from `/api/vhosts`

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Fixes: #6686

This PR adds per vhost message stats metrics.

Message stats:
  -  ack
  -  confirm
  -  deliver
  -  get
  -  get_no_ack
  -  publish
  -  redeliver
  -  return_unroutable

Modules gathers these metrics from `/api/vhosts` endpoint.

##### Component Name

[`/collectors/python.d.plugin/rabbitmq`](https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/elasticsearch/rabbitmq.chart.py)

### CODE REVIEW

**Comment by Wing924** [comment]
output metrics:
```
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="ack",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="confirm",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="deliver",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="get",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="get_no_ack",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="publish",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="redeliver",family="vhosts"}	0
netdata_rabbitmq_vhost_messages_msg_persec_average{chart="rabbitmq_local.vhost___message_stats",dimension="return_unroutable",family="vhosts"}
```

It's hard to get vhost name.
How about change family to vhost name?

File: collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py [hunk 1]
```diff
@@ -111,6 +123,49 @@
 }
 
 
+def vhost_chart_template(name):
+    order = [
+        'vhost_{0}_message_stats'.format(name),
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py]: Mm, probably you are right, ill remove it from `chartID`

**Comment by ilyam8** [comment]
@Wing924 fixed

File: collectors/python.d.plugin/rabbitmq/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # rabbitmq
 
-Module monitor rabbitmq performance and health metrics.
+Module monitor [`RabbitMQ`](https://www.rabbitmq.com/) performance and health metrics.
```
Comment (joelhans) [inline collectors/python.d.plugin/rabbitmq/README.md]:
```suggestion
This module monitors [RabbitMQ](https://www.rabbitmq.com/) performance and health metrics.
```

File: collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py [hunk 3]
```diff
@@ -111,6 +123,58 @@
...
+        return name[1:]
+    return name
+
+
+def vhost_chart_template(name):
+    name = normalize_vhost_name(name)
+    order = [
+        'vhost_{0}_message_stats'.format(name),
+    ]
+    family = 'vhost {0}'.format(name)
```
Comment (Wing924) [inline collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py]: family should use raw name like disk volume

**Pull request merged**